<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>1271</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="GraphicsLayoutWidget" name="graph_spectre"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="GraphicsLayoutWidget" name="graph_rgb"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QSpinBox" name="sb_lowRed">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 121, 121);</string>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRangeSlider" name="dSlider_red">
             <property name="maximum">
              <number>1024</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_highRed">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 121, 121);</string>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>341</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSpinBox" name="sb_lowGreen">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(125, 231, 106);</string>
             </property>
             <property name="maximum">
              <number>1024</number>
             </property>
             <property name="value">
              <number>342</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRangeSlider" name="dSlider_green">
             <property name="maximum">
              <number>1024</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_highGreen">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(125, 231, 106);</string>
             </property>
             <property name="maximum">
              <number>1024</number>
             </property>
             <property name="value">
              <number>682</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSpinBox" name="sb_lowBlue">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(138, 173, 255);</string>
             </property>
             <property name="maximum">
              <number>1024</number>
             </property>
             <property name="value">
              <number>683</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRangeSlider" name="dSlider_blue">
             <property name="maximum">
              <number>1024</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_highBlue">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(138, 173, 255);</string>
             </property>
             <property name="maximum">
              <number>1024</number>
             </property>
             <property name="value">
              <number>1024</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="cmb_wave">
           <item>
            <property name="text">
             <string>WaveNumber</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WaveLength</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QComboBox" name="cmb_set_maximum">
             <item>
              <property name="text">
               <string>Set to Global Maximum</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Set to Local Maximum</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pb_search">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QRangeSlider</class>
   <extends>QSlider</extends>
   <header>/gui/widgets/QRangeSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
